.github/workflows/test-all-packages.yml

```diff
@@ -78,6 +78,43 @@ jobs:
     - name: lint check
       run: yarn lint-check
 
+  benchmark:
```

Comment:
Should this be its own workflow file?

Reply:
You asked the same question on the other (now dead) PR :-)

The answer is that would be more expensive because it's dependent on the build step.

Reply:
Hahaha, I'm nothing if not consistent! Ok, that's fair.

Reply:
My instinct is to put this in its own workflow file too, and have it do its own build (however much of a "build" is necessary for this benchmark: probably not as much as the full unit tests). We should glance at how long the `build` step which this relies upon actually takes: if it's short enough, or if we can make a subset that is, it might be worth spending the extra CPU seconds in exchange for a more tidy separation of "PR acceptance tests" vs "periodic measurement tasks".

But we should try it this way and see what happens. I think we'll see an extra line of noise in the CI results for each PR (showing this job being skipped).